config/application.rb

```diff
@@ -175,12 +175,8 @@ def try_config_for(*args)
     config.three_scale.sandbox_proxy = ActiveSupport::OrderedOptions.new
     config.three_scale.sandbox_proxy.merge!(config_for(:sandbox_proxy).symbolize_keys)
 
-    config.three_scale.web_analytics = ActiveSupport::OrderedOptions.new
```

Comment:
On line 14 there is a reference to `preview` environment. I think we want it removed.